deploy/examples/import-external-cluster.sh

```diff
@@ -152,6 +171,55 @@ function importRGWAdminOpsUser() {
   --from-literal=secretKey="$RGW_ADMIN_OPS_USER_SECRET_KEY"
 }
 
+function createRBDStorageClass() {
+cat << eof | kubectl create -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: $RBD_STORAGE_CLASS_NAME
```

Comment:
Is the second cluster should keep the same StorageClassName and Namespace,
Or it should be different

Reply:
we can keep the same if we are deploying in a different kubernetes cluster